playground/src/App.tsx

```diff
@@ -142,46 +213,75 @@ export default function App() {
       responseSectionParent.append(div);
     }
 
+    const toolbar = document.getElementsByClassName('graphiql-toolbar')[0] as any as HTMLDivElement;
+
+    if (toolbar) {
+      const toggleClientValidation = document.createElement('div');
+      toggleClientValidation.id = 'toggle-client-validation';
+      toolbar.append(toggleClientValidation);
+    }
+
     setIsMounted(true);
   }, [isMounted]);
 
+  useEffect(() => {
+    const getSchema = async () => {
```

Comment:
Are we not already loading the Schema from the backend using an API?

Reply:
This is in router playground. There is a race when accessing the one that graphiql provides in the callback. So doing it separately.